life/updated/02-life.adoc

```diff
@@ -358,137 +358,144 @@ He took me by the hand, and led me by a way I knew not. He made darkness light b
-a hundred miles off, and with such a plain testimony as the Lord did require of me;
-which made me go bowed down many months under the exercise of it;
-and oftentimes strove against it.
-I could get no rest,
-but in giving up to obey the Lord in all things that he required of me;
-and though it seemed hard and strange to me, yet the Lord made hard things easy,
-according to his promise to me, when I was going from my children,
-and knew not but my life might be required for my testimony, it was so plain;
-and when I looked upon my children, my heart yearned towards them.
+a hundred miles off, and with such a plain testimony as the Lord did require of me.
+This made me go bowed down many months under the exercise of it,
+and oftentimes I strove against it.
+I could get no rest
+except in giving up to obey the Lord in all things that He required of me.
+And though it seemed hard and strange to me, yet the Lord made hard things easy,
+according to His promise to me.
+When I was going from my children,
+and knew not but my life might be required for my testimony,
+the will of the Lord appeared so plain;
+yet when I looked upon my children, my heart yearned towards them.
 These words ran through me, "`If you can believe, you shall see all things accomplished,
 and you shall return in peace, and your reward shall be with you.`"
-Forever blessed be the name and power of the Lord, he sustained me in my journey,
-gave me strength to do his will, and afforded me his living presence to accompany me,
+Forever blessed be the name and power of the Lord, for He sustained me in my journey,
+gave me strength to do His will, and afforded me His living presence to accompany me,
 which is the greatest comfort that can be enjoyed.
 
 [.offset]
-This was my testimony to King Charles II., in the eleventh month, of the year 1670.
+This was my testimony to King Charles II., in the eleventh month of the year 1670.
+
+[.embedded-content-document.address]
+--
 
-"`This is unto you, O king:
+This is unto you, O king.
 Hear what the Lord has committed unto my charge concerning you.
 As you have been the cause of making many desolate, so will the Lord lay you desolate;
 and as many as have been the cause of persecuting and
 shedding the blood of my dear children,
 in the day when I call all to an account, I will plead with them, says the Lord.
 Therefore hear and fear the Lord God of heaven and earth,
-for of his righteous judgments all shall be made partakers;
-from the king that sits upon the throne, to the beggar upon the dunghill.`"
+for of His righteous judgments all shall be made partakers;
+from the king that sits upon the throne, to the beggar upon the dunghill.
+
+--
 
 This testimony I delivered into his hands, with these words, "`Hear, oh king,
 and fear the Lord God of heaven and earth.`"
-I can truly say, that the dread of the most high God was upon me, which made me tremble,
-and great agony was over my spirit; insomuch that paleness came in his face,
```

Comment:
I think we can leave _insomuch_ in most cases. I think people generally understand it.